### PR TITLE
Support retrieve connection string from ARM

### PR DESCRIPTION
**Pull request by sophiaso**
## Description
When connection string not configured, will try to retrieve connection string from ARM.

### CODE REVIEW

File: codestyle/checkstyle/checkstyle.xml [hunk 1]
```diff
@@ -165,7 +165,7 @@ page at http://checkstyle.sourceforge.net/config.html -->
 
             <property name="ignorePattern"
                       value="${com.puppycrawl.tools.checkstyle.checks.sizes.LineLength.ignorePattern}"
-                      default="^(package .*;\s*)|(import .*;\s*)|( *\* *https?://.*)$"/>
+                      default="^(package .*;\s*)|(import .*;\s*)|( *\* *https?://.*)|( *.*https?://.*)$"/>
```
Comment (warrenzhu25) [inline codestyle/checkstyle/checkstyle.xml]: What's the purpose of this change? And you might need to doc this in commit message.

Comment (sophiaso) [reply codestyle/checkstyle/checkstyle.xml]: Want to ignore comments starting with `*` and contains `http(s)://` link, in order to show in one line.
